src/libraries/System.Data.Odbc/src/Common/System/Data/Common/AdapterUtil.Odbc.cs

```diff
@@ -656,7 +656,6 @@ internal static int StringLength(string inputString)
             return ((null != inputString) ? inputString.Length : 0);
         }
 
-        [ReliabilityContract(Consistency.WillNotCorruptState, Cer.MayFail)]
```

Comment:
This project targets netstandard + netframework.

Reply:
The file is in an ItemGroup that only targets NetCoreApp.